### PR TITLE
Added knowledge base link to the navbar

### PR DESCRIPTION
**Pull request by monzanifabio**
Added the Notion link to the landing page nav component

### CODE REVIEW

File: lib/components/Nav.jsx [hunk 1]
```diff
@@ -18,6 +18,26 @@ export const Nav = (props) => {
...
+        shallow
+      >
+        <a
+          target='_blank'
+          className={classnames(
+            'mr-3',
+            navLinkClasses,
+            {
+              'text-white hover:text-highlight-2': !developersPage,
+              'text-highlight-2 hover:text-highlight-2': developersPage
```
Comment (chuckbergeron) [inline lib/components/Nav.jsx:33]: These class differences only apply to internal links (ie. links on https://pooltogether.com). If we're linking out to an external site (notion in this case) we can simply use these two classes: `text-white hover:text-highlight-2`

File: lib/components/Nav.jsx [hunk 2]
```diff
@@ -18,6 +18,26 @@ export const Nav = (props) => {
     <nav
       className='justify-end items-center hidden sm:flex w-2/3'
     >
+      <Link
```
Comment (chuckbergeron) [inline lib/components/Nav.jsx:21]: The next.js Link component isn't necessary here as that helps with the SPA routing. We can simply use a regular `<a>` tag with the `href` attribute.

File: lib/components/Nav.jsx [hunk 3]
```diff
@@ -18,6 +18,26 @@ export const Nav = (props) => {
...
+          className={classnames(
+            'mr-3',
+            navLinkClasses,
+            {
+              'text-white hover:text-highlight-2': !developersPage,
+              'text-highlight-2 hover:text-highlight-2': developersPage
+            }
+          )}
+        >
+          {t('knowledge base')}
```
Comment (chuckbergeron) [inline lib/components/Nav.jsx:37]: These `t()` function calls are for the Locize backend which we don't use on pooltogether.com yet! It's all in plain English.

File: lib/components/Nav.jsx [hunk 4]
```diff
@@ -18,6 +18,26 @@ export const Nav = (props) => {
     <nav
       className='justify-end items-center hidden sm:flex w-2/3'
     >
+      <Link
+        href='https://www.notion.so/PoolTogether-Knowledge-Base-fa721ccefa3242eaabd125a8415acd27'
+        as='https://www.notion.so/PoolTogether-Knowledge-Base-fa721ccefa3242eaabd125a8415acd27'
+        shallow
+      >
+        <a
```
Comment (chuckbergeron) [inline lib/components/Nav.jsx:26]: I believe this would be better suited for the <Footer />. Personally I try to leave the top nav items as internal links so the navigation is always consistent (ie. you click "Developers" and the nav at the top stays the same. The footer is a better place for external links to other apps/sites. And/or in the content of the pages itself. I believe @Lililashka is working on a redesign of this site right now, though! Perhaps it would be best to include her in discussions around having links to the new Knowledge Base integrated into the landing site.

**Review by chuckbergeron — CHANGES_REQUESTED** [review]

File: lib/components/Nav.jsx (hunk 4 above)
Comment (monzanifabio) [reply lib/components/Nav.jsx:26]: Hey @chuckbergeron thanks for the feedback.
Still not very practical with React.
We can have a chat with @Lililashka about the placement but we could put it in the footer for now.

**Comment by chuckbergeron** [comment]
Added to footer! @monzanifabio https://pooltogether.com
